models/maker/maker_accounting.sql

```diff
@@ -0,0 +1,839 @@
+    WHERE u = '0xa950524441892a31ebddf91d3ceefa04bf454466' -- Vow
+    AND v NOT IN ('0x197e90f9fad81970ba7976f33cbd77088e5d7cf7', '0xbe8e3e3618f7474f8cb1d074a26affef007e98fb', '0x2cc583c0aacdac9e23cb601fda8f1a0c56cdcb71', '0xa4c22f0e25c6630b2017979acf1f865e94695c4b') -- dsr, opex
+    AND call_success
+    GROUP BY 1,2
+    
+    UNION ALL
+    
+    SELECT call_block_time ts
+    , call_tx_hash hash
+    , 21120 AS code
+    , SUM(rad)/POW(10,45) AS value --increased liability
+    FROM {{ source('maker_ethereum','vat_call_suck') }}
+    WHERE u = '0xa950524441892a31ebddf91d3ceefa04bf454466' -- Vow
+    AND v NOT IN ('0x197e90f9fad81970ba7976f33cbd77088e5d7cf7', '0xbe8e3e3618f7474f8cb1d074a26affef007e98fb', '0x2cc583c0aacdac9e23cb601fda8f1a0c56cdcb71', '0xa4c22f0e25c6630b2017979acf1f865e94695c4b') -- dsr, opex
+    AND call_success
+    GROUP BY 1,2
+), sin_inflows AS 
+(
+    SELECT call_block_time ts
+    , call_tx_hash hash
+    , 31510 AS code
+    , SUM(rad)/POW(10,45) AS value --increased equity
+    FROM {{ source('maker_ethereum','vat_call_suck') }}
+    WHERE v = '0xa950524441892a31ebddf91d3ceefa04bf454466' -- Vow
+    AND call_success
+    GROUP BY 1,2
+    
+    UNION ALL
+    
+    SELECT call_block_time ts
+    , call_tx_hash hash
+    , 21120 AS code
+    , -SUM(rad)/POW(10,45) AS value --decreased liability
+    FROM {{ source('maker_ethereum','vat_call_suck') }}
+    WHERE v = '0xa950524441892a31ebddf91d3ceefa04bf454466' -- Vow
+    AND call_success
+    GROUP BY 1,2
+), dsr_flows_preunioned AS
+(
+    SELECT call_block_time ts
+    , call_tx_hash hash
+    , -rad/POW(10,45) AS dsr_flow
+    FROM {{ source('maker_ethereum','vat_call_move') }} m
+    WHERE call_success
+    AND src = '0x197e90f9fad81970ba7976f33cbd77088e5d7cf7'
+    
+    UNION ALL
+    
+    SELECT call_block_time ts
+    , call_tx_hash hash
+    , rad/POW(10,45) AS dsr_flow
+    FROM {{ source('maker_ethereum','vat_call_move') }}  m
+    WHERE call_success
+    AND dst = '0x197e90f9fad81970ba7976f33cbd77088e5d7cf7'
+), dsr_flows AS
+(
+    SELECT ts
+    , hash
+    , 21110 AS code
+    , dsr_flow AS value -- positive dsr flow increases interest-bearing dai liability
+    FROM dsr_flows_preunioned
+    
+    UNION ALL
+    
+    SELECT ts
+    , hash
+    , 21120 AS code
+    , -dsr_flow AS value -- positive dsr flow decreases non-interest-bearing dai liability
+    FROM dsr_flows_preunioned
+), treasury_flows_preunioned AS
+(
+    SELECT evt_block_time ts
+    , evt_tx_hash hash
+    , token
+    , SUM(value/pow(10,decimals)) AS value
+    FROM {{ source('erc20_ethereum','evt_transfer') }}
+    JOIN treasury_erc20s
+    USING (contract_address)
+    WHERE `to` = '0xbe8e3e3618f7474f8cb1d074a26affef007e98fb'
+    GROUP BY 1,2,3
+    
+    UNION ALL
+    
+    SELECT evt_block_time ts
+    , evt_tx_hash hash
+    , token
+    , -SUM(value/pow(10,decimals)) AS value
+    FROM {{ source('erc20_ethereum','evt_transfer') }}
+    JOIN treasury_erc20s
```

Comment:
Hi @SebVentures, this model is a large model. Since the `evt_transfer` table is used here, would you mind if I modify this model into incremental view(this will preserve all model functionality)?

Reply:
Sure, no problem. We might change the query as we go along and it wasn't clear in the docs how to reset such a table.
Happy to learn!

Reply:
The table/view will update entries based on unique keys. During incremental merging stage, new records will match with old records and then merge. Values with new/not existed unique keys will be appended to the table, while values with existed unique keys will replace the corresponding old values.